server/src/controllers/CompanyController.js

```diff
@@ -42,22 +42,26 @@ const createCompany = async (req, res) => {
     }
 };
 
-const getCompanies = async (req, res) => {
+const getCompanies = async (req, res, next) => {
     try {
         const companies = await Company.find();
+        // res.locals.cache = companies;
+        // next();
```

Comment:
Please remove comments if not needed. Also, if next is not needed we can remove it.